extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchBuildBufferAggregator.java

```diff
@@ -42,16 +43,13 @@
 public class HllSketchBuildBufferAggregator implements BufferAggregator
 {
 
-  /** for locking per buffer position (power of 2 to make index computation faster) */
-  private static final int NUM_STRIPES = 64;
-
   private final ColumnValueSelector<Object> selector;
   private final int lgK;
   private final TgtHllType tgtHllType;
   private final int size;
   private final IdentityHashMap<ByteBuffer, WritableMemory> memCache = new IdentityHashMap<>();
   private final IdentityHashMap<ByteBuffer, Int2ObjectMap<HllSketch>> sketchCache = new IdentityHashMap<>();
```

Comment:
It seems that this field is not really a "cache". The corresponding field in `SketchBufferAggregator` is called "unions". Maybe call it "sketches".